app/src/ui/sign-in/sign-in.tsx

```diff
@@ -203,10 +203,16 @@ export class SignIn extends React.Component<ISignInProps, ISignInState> {
             value={this.state.password}
             type="password"
             onValueChanged={this.onPasswordChanged}
-            labelLinkText="Forgot password?"
-            labelLinkUri={state.forgotPasswordUrl}
           />
         </Row>
+        <Row>
+          <LinkButton
+            className="forgot-password-link-sign-in"
+            uri={state.forgotPasswordUrl}
+          >
+            Forgot password
```

Comment:
Seems like the question mark got dropped when moving this, could we bring it back please?

Reply:
Actually, I did it on purpose to be on par with [@app/src/ui/lib/authentication-form.tsx#L124](https://github.com/desktop/desktop/blob/master/app/src/ui/lib/authentication-form.tsx#L124). Should I add one there, too?

Reply:
Yes, I'd really appreciate that!